backend/howtheyvote/models/vote.py

```diff
@@ -84,13 +90,15 @@ class Vote(BaseWithId):
     term: Mapped[int] = mapped_column(sa.Integer)
     order: Mapped[int] = mapped_column(sa.Integer)
     title: Mapped[str | None] = mapped_column(sa.Unicode)
+    dlv_title: Mapped[str | None] = mapped_column(sa.Unicode)
```

Comment:
DLV seems to be the abbreviation for a system the EP uses for this data sources. The name is not great, but I couldn’t come up with something else, and it isn’t exposed in the export and API.

Reply:
For future reference: Each block/group of votes has a DLV ID, and this is the title of the block. It might also be an abbreviation for something else, not the IT system.